### PR TITLE
Fixed deprecation warning in the dll:build script

### LINKED ISSUE #10980: Deprecation warnings in webpack 5
## 📝 Provide detailed reproduction steps (if any)

A few tasks may print logs related to API deprecation.

### `dll:build`

```
(node:27547) [DEP_WEBPACK_COMPILATION_OPTIMIZE_CHUNK_ASSETS] DeprecationWarning: optimizeChunkAssets is deprecated (use Compilation.hooks.processAssets instead and use one of Compilation.PROCESS_ASSETS_STAGE_* as stage option)
(Use `node --trace-deprecation ...` to show where the warning was created)
```

Let's see what we can do to remove them.

### PR DESCRIPTION
**Pull request by przemyslaw-zan**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/contributing/git-commit-message-convention.html))

Internal: Fixed deprecation warning in the `dll:build` script. Closes #10980.

---

### Additional information

_For example – encountered issues, assumptions you had to make, other affected tickets, etc._

### CODE REVIEW

**Comment by przemyslaw-zan** [comment]
It looks like my reference to the stale issue https://github.com/levp/wrapper-webpack-plugin/issues/11 was noticed, and PR with fixes was proposed:
- https://github.com/levp/wrapper-webpack-plugin/pull/22

File: scripts/dll/webpackFooterPlugin.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+	 * @param {String} footer Text that will be appended to an output file.
+	 */
+	constructor( footer ) {
+		validate( { type: 'string' }, footer, { name: FooterPlugin.name } );
+
+		this.footer = footer;
+	}
+
+	apply( compiler ) {
+		const { ConcatSource } = compiler.webpack.sources;
```
Comment (psmyrek) [inline scripts/dll/webpackFooterPlugin.js:23]: This works in webpack v5+. Should we care about webpack v4?

**Review by psmyrek — APPROVED** [review]
LGTM 🎉 

The footer in DLL builds is added correctly:

obraz

File: scripts/dll/webpackFooterPlugin.js (hunk 1 above)
Comment (pomek) [reply scripts/dll/webpackFooterPlugin.js:23]: We don't. It's a development script used only by us.

**Review by pomek — APPROVED** [review]
LGTM.

`#master`

```
yarn webpack --config scripts/dll/webpack.config.dll.js
yarn run v1.22.19
$ /home/pomek/Projects/ckeditor/ckeditor5/node_modules/.bin/webpack --config scripts/dll/webpack.config.dll.js
(node:242414) [DEP_WEBPACK_COMPILATION_OPTIMIZE_CHUNK_ASSETS] DeprecationWarning: optimizeChunkAssets is deprecated (use Compilation.hooks.processAssets instead and use one of Compilation.PROCESS_ASSETS_STAGE_* as stage option)
(Use `node --trace-deprecation ...` to show where the warning was created)
```

`#ck/10980`

```
 yarn webpack --config scripts/dll/webpack.config.dll.js
yarn run v1.22.19
$ /home/pomek/Projects/ckeditor/ckeditor5/node_modules/.bin/webpack --config scripts/dll/webpack.config.dll.js
asset ckeditor5-dll.js 713 KiB [emitted] [minimized] [big] (name: main)
...
```

`window.CKEditor5`

image

**Pull request merged**